src/applications/vaos/appointment-list/components/AppointmentsPage/AppointmentsPageV2.jsx

```diff
@@ -139,15 +118,7 @@ function AppointmentsPageV2({
         )}
       />
 
-      {showScheduleButton && (
-        <div className="vads-u-margin-bottom--4">
-          <ScheduleNewAppointmentRadioButtons
-            showCheetahScheduleButton={canUseVaccineFlow}
-            startNewAppointmentFlow={startNewAppointmentFlow}
-            startNewVaccineFlow={startNewVaccineFlow}
-          />
-        </div>
-      )}
+      {showScheduleButton && <ScheduleNewAppointmentRadioButtons />}
```

Comment:
I didn't do much on this V2 page other than make sure it worked. The styling is the same as the regular homepage, and I assume will need to be tweaked later.